mobile/test/common/integration/client_integration_test.cc

```diff
@@ -71,6 +73,11 @@ TEST_P(ClientIntegrationTest, Basic) {
   ASSERT_EQ(cc_.on_complete_received_byte_count, 67);
 }
 
+TEST_P(ClientIntegrationTest, Basic) {
+  basicTest();
+  TearDown();
```

Comment:
Isn't TearDown() called automatically by the test infrastructure? I'm not familiar with it being called explicitly.

Reply:
it does. this is a temporay holdover from me trying to add loging at which point we ned to explictly destruct before loging macros go away. that code wasn't finished but I'd mildly prefer to leave this as-is to regresion test teardown can be called multiple times